certification/internal/policy/container/has_unique_tag.go

```diff
@@ -19,7 +19,8 @@ type HasUniqueTagCheck struct {
 }
 
 func (p *HasUniqueTagCheck) Validate(imgRef certification.ImageReference) (bool, error) {
-	tags, err := p.getDataToValidate(imgRef.ImageURI)
+	uriWithoutTags := strings.Split(imgRef.ImageURI, ":")[0]
```

Comment:
I think this won't work if the image is pinned with the SHA tag.

Reply:
I thought we had some logic for extracting the basename of the image (e.g. without tag or digest) at some point in time.

Reply:
It was part of the SkopeoEngine logic that has been removed.